### PR TITLE
Add another TLDR

### PR DESCRIPTION
**Pull request by joulev**

### CODE REVIEW

**Comment by joulev** [comment]
Woah how did you manage to do this?

**Comment by ChristianIvicevic** [comment]
I added a fake environment variable in Vercel that prevents deployments from forks. I am just afraid that it blocks EVERY future push since it could technically introduce additional ways to leak environment variables. We'll see once you push additional changes.

File: apps/web/src/app/page.tsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 import type { ComponentProps, ReactNode } from 'react'
+import { ChevronsDown } from 'lucide-react'
 import { ThemeSwitcher } from '@/app/components.client'
 import { cn } from '@/lib/utils'
 
-function H1({ children }: { readonly children: ReactNode }) {
-  return <h1 className="scroll-m-20 text-4xl font-extrabold tracking-tight lg:text-5xl">{children}</h1>
+function H1({ id, children }: { id?: string; readonly children: ReactNode }) {
```
Comment (ChristianIvicevic) [inline apps/web/src/app/page.tsx:6]: I have a habit of marking all the things `readonly`, could you apply that to `id` as well?

File: apps/web/src/app/page.tsx [hunk 2]
```diff
@@ -42,19 +51,63 @@ function Code({ children }: { readonly children: ReactNode }) {
...
   return (
     <div className="container relative">
-      <div className="flex items-center justify-between py-12">
-        <H1>Adios Axios 👋</H1>
+      <div className="fixed bottom-8 right-8">
         <ThemeSwitcher />
       </div>
-      <div className="mb-12 flex flex-col gap-8">
-        <section>
+      <div className="flex min-h-screen flex-col justify-between" style={{ minHeight: '100svh' }}>
```
Comment (ChristianIvicevic) [inline apps/web/src/app/page.tsx:57]: Isn't `min-h-screen` and `min-height: 100svh` conflicting? Shouldn't that just be `min-h-[100svh]`?

**Review by ChristianIvicevic — CHANGES_REQUESTED** [review]

File: apps/web/src/app/page.tsx (hunk 2 above)
Comment (joulev) [reply apps/web/src/app/page.tsx:57]: IIRC `svh` is not supported in all browsers yet so I added that `min-h-screen` as fallback. Lemme check again.

Comment (ChristianIvicevic) [reply apps/web/src/app/page.tsx:57]: It looks like it's "good enough" unless I'm mistaken: https://caniuse.com/mdn-css_types_length_viewport_percentage_units_small

Comment (joulev) [reply apps/web/src/app/page.tsx:57]: Oh it's supported everywhere now, nice ✨

**Comment by ChristianIvicevic** [comment]
Thanks for the contribution - really great addition 🎉

**Comment by joulev** [comment]
Oops, forgot to reload the tab after pushing. Thanks for merging!

**Pull request merged**